conf/upf.json

```diff
@@ -42,10 +42,21 @@
     "": "Whether to enable Network Token Functions",
     "enable_ntf": false,
 
+    "": "Whether to enable P4Runtime feature",
+    "enable_p4rt": false,
+    
     "": "Control plane controller settings",
     "cpiface": {
-        "nb_dst_ip": "172.17.0.1",
-        "" : "nb_dst_ip: CPHostname",
-        "hostname": "ngic-skx2"
+	"nb_dst_ip": "172.17.0.1",
+	"" : "nb_dst_ip: CPHostname",
+	"hostname": "ngic-skx2"
+    },
+    
+    "": "pfcp agent settings",
+    "pfcpiface": {
+	"access_ip": "172.17.0.1/32",
+    "p4rtc_server": "onos",
+    "p4rtc_port": 51001,
+    "ue_ip_pool": "10.250.0.0/24"
```

Comment:
This one is tricky. We have been using `cpiface` json block for both zmq-based and pfcp-based control planes. Which means that there are some fields in the `cpiface` block that will be used for communicating with the N4 northbound interface for pfcp communication. We can either combine the 2 blocks, or rename the `pfcpiface` block to a more specific name.

Reply:
Should we name to p4rtc block ?

Reply:
Sure.

Reply:
fixed